pkg/ingress/parser.go

```diff
@@ -274,7 +275,7 @@ func (c *Controller) generateConfig() error {
 			if len(eps) > 0 {
 				def := &haproxy.TCPService{
 					SharedInfo:   si,
-					FrontendName: fmt.Sprintf("tcp-%s", rule.TCP.Port),
+					FrontendName: fmt.Sprintf("tcp-%s", rule.TCP.Port.String()),
```

Comment:
yes we do need to call String(), before when we are not calling `String()` the template generated have values like `*intstr{some random value}`